code/python/pipelines-workshop/cdk.json

```diff
@@ -1,8 +1,6 @@
 {
   "app": "python3 app.py",
   "context": {
-    "@aws-cdk/core:enableStackNameDuplicates": "true",
-    "aws-cdk:enableDiffNoFail": "true",
-    "@aws-cdk/core:stackRelativeExports": "true"
```

Comment:
None of these are required however `@aws-cdk/core:newStyleStackSynthesis` was missing here but is required in order for this code to `synth`